### PR TITLE
Feat: Add js selectors for h1 title state

### PR DESCRIPTION
**Pull request by djimrangarleita**
It's cool but try adding at least a console.log to the file script.js first

### CODE REVIEW

**Review by djimrangarleita — COMMENTED** [review]
Please consider adding a console.log to the file

**Comment by djimrangarleita** [comment]
We may close this pull request.

**Review by mossosouk-sarl — CHANGES_REQUESTED** [review]
Reviewed and may need improvements

File: script.js [hunk 1]
```diff
@@ -0,0 +1 @@
+consle.log("Bonjour")
```
Comment (djimrangarleita) [inline script.js]: check the spelling of console

**Review by djimrangarleita — COMMENTED** [review]
Please check the spelling

**Review by djimrangarleita — COMMENTED** [review]
Closes#2

**Review by mossosouk-sarl — APPROVED** [review]
Well done 👍🏽

**Pull request merged**